lib/ansible/modules/packaging/os/yum.py

```diff
@@ -279,32 +281,34 @@ def yum_base(conf_file=None, installroot='/'):
 
     return my
 
-def ensure_yum_utils(module):
```

Comment:
removing this module due to following reasons.

1) module should fail when yum_utils is not installed, currently, it will try to install it without being instructed to do so.